modules/installation-azure-user-defined-routing.adoc

```diff
@@ -0,0 +1,84 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_azure/installing-azure-private.adoc
+
+[id="installation-azure-user-defined-routing_{context}"]
+= User-defined outbound routing
+
+In {product-title}, you can choose your own outbound routing for a cluster to
+connect to the internet. This allows you to skip the creation of public IP
```

Comment:
You have both uppercase and lowercase instances of the internet throughout this PR. Should be consistent.